### PR TITLE
Authentication pages

### PR DESCRIPTION
**Pull request by foolhardy21**
This PR has:
- Login Page.
- Signup Page.

Feel free to review. Thanks.

### CODE REVIEW

**Review by partha8 — COMMENTED** [review]
About the signup page, there needs to be a field for first name, last name, email, set password and confirm password. A checkbox is also needed which hides/shows password. Its in the PRD apparently and I missed it too.

**Comment by foolhardy21** [comment]
> About the signup page, there needs to be a field for first name, last name, email, set password and confirm password. A checkbox is also needed which hides/shows password. Its in the PRD apparently and I missed it too.

Thanks for mentioning @partha8 . I have added the input fields and password toggle functionality.

**Review by partha8 — APPROVED** [review]

File: script.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const togglePassword = document.querySelector('#toggle-pass')
+const passwordInput = document.querySelector('#pass-input')
+const confirmPasswordInput = document.querySelector('#cpass-input')
+
+togglePassword.addEventListener('change', () => {
+    if(passwordInput.type === 'password') {
+        passwordInput.type = 'text'
+        confirmPasswordInput.type = 'text' 
+    } else if(passwordInput.type === 'text') {
```
Comment (vrinda-mahajan) [inline script.js]: As there are only two conditions, you can avoid the else if block and use an else block instead.

File: signup.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                </label>
+                <input type='password' id='cpass-input' class="'margin-top-sm no-outline padding-sm border-thin" />
+                <div class='flex-horizontal-end flex-vertical-center margin-top-md margin-btm-sm c-darkgray'>
+                    <input type='checkbox' id='toggle-pass' />
+                    <label for='toggle-pass' class='text-capital margin-left-sm'>
+                        show password
+                    </label>
+                </div>
+                <div class='flex-horizontal-start flex-vertical-center margin-top-sm margin-btm-lg c-darkgray'>
+                    <input type='radio' id='terms-check' />
```
Comment (vrinda-mahajan) [inline signup.html]: It is good to use a checkbox instead of a radio button here.

**Review by vrinda-mahajan — CHANGES_REQUESTED** [review]
Just added a few suggestions. Everything else looks good.

**Comment by foolhardy21** [comment]
thanks @vrinda-mahajan. I have made the changes.

**Review by vrinda-mahajan — APPROVED** [review]

File: script.js (hunk 1 above)
Comment (foolhardy21) [reply script.js]: code update.

File: signup.html (hunk 2 above)
Comment (foolhardy21) [reply signup.html]: radio replaced with checkbox

**Pull request merged**